src/dsp_tools/models/exceptions.py

```diff
@@ -42,29 +42,6 @@ def __init__(self, custom_msg: str | None = None, keep_default_msg: bool = True)
                 super().__init__(default_msg)
 
 
-class RetryError(BaseError):
```

Comment:
in `entry_point.py`, we have a reference to this error class (in a docstring), which should also be removed